docs/registry/index.rst

```diff
@@ -276,29 +276,190 @@ For an example of how this might look like, see `GAVO's plate tutorial`_
 
 .. _GAVO's plate tutorial: http://docs.g-vo.org/gavo_plates.pdf
 
+More examples
+-------------
+
+Discover archives
+^^^^^^^^^^^^^^^^^
+
+You can use the registry ``search`` method (or the ``regsearch`` function) 
+to discover archives that may have x-ray images and then query those archives 
+to find what x-ray images that have of CasA. For the arguments you will 
+enter ``'image'`` for the service type and ``'x-ray'`` for the waveband. 
+The position is provided by the Astropy library.
+
+The query returns a :py:class:`~pyvo.registry.regtap.RegistryResults` object 
+which is a container holding a table of matching services. In this example 
+it returns 33 matching services.
+
+.. doctest-remote-data::
+
+  >>> import pyvo as vo
+  >>> from astropy.coordinates import SkyCoord
+
+  >>> import warnings
+  >>> warnings.filterwarnings('ignore', module="astropy.io.votable.*")
+
+  >>> archives = vo.regsearch(servicetype='image', waveband='x-ray') 
+  >>> pos = SkyCoord.from_name('Cas A')
+  >>> len(archives)   # doctest: +IGNORE_OUTPUT
+  33
```

Comment:
For example, this example generates a CI test failure, as due to the empty lines it is picked up as 3 separate examples and remote-data only added for the first one.

I would suggest to add `>>>` in the empty lines as well, so eventually everything stays as one example.  
```suggestion
.. doctest-remote-data::

  >>> import pyvo as vo
  >>> from astropy.coordinates import SkyCoord
  >>>
  >>> import warnings
  >>> warnings.filterwarnings('ignore', module="astropy.io.votable.*")
  >>>
  >>> archives = vo.regsearch(servicetype='image', waveband='x-ray') 
  >>> pos = SkyCoord.from_name('Cas A')
  >>> len(archives)   # doctest: +IGNORE_OUTPUT
  33
```

Reply:
Thank you so much! I was wondering why it did not work since the code seem to be fine and `docktest` runs pretty well locally. Now the issue is fixed.